server/chat-plugins/friends.ts

```diff
@@ -156,7 +156,10 @@ export const Friends = new class {
 	checkCanUse(context: Chat.CommandContext | Chat.PageContext) {
 		const user = context.user;
 		if (!user.autoconfirmed) {
-			throw new Chat.ErrorMessage(context.tr`You must be autoconfirmed to use the friends feature.`);
+			throw new Chat.ErrorMessage(
+				context.tr`To use the friends feature you must be autoconfirmed, which means being registered ` +
+				`for at least one week and winning one rated game.`
```

Comment:
Actually I think this is wrong for translating.

```suggestion
				context.tr`To use the friends feature you must be autoconfirmed, which means being registered for at least one week and winning one rated game.`
```

Reply:
Issue is without splitting the line is it fails the lint check for 120 character limit

Reply:
I don't think you should resolve this without a solution. In theory the lint check should be able to handle this. If not, I just need to port over the client lint code which has some updates that I think includes this case.